proto/launch/params.proto

```diff
@@ -11,4 +11,5 @@ message Params {
 
   uint64 minLaunchTime = 1 [(gogoproto.moretags) = "yaml:\"min_launch_time\""];
   uint64 maxLaunchTime = 2 [(gogoproto.moretags) = "yaml:\"max_launch_time\""];
+  uint64 revertDelay = 3   [(gogoproto.moretags) = "yaml:\"revert_delay\""];
```

Comment:
I think we can use `int64` instead since the type is manipulated with UNIX timestamp, we can easily ensure the value is positive from the validate method while `int64` conversion comes with some vulnerabilities

If we go this way, we will do the change as well for `minLaunchTime` and `maxLaunchTime`

Reply:
Ok, I'll add a check that values are greater than or equal to zero as well.